### DISCUSSION TITLE
livewire and selectize.js

### DISCUSSION THREAD

**Discussion by asadghazanfary** [livewire/livewire#3983] [1. Help]
hello guys

Does anyone have experience working with selectize.js in livewire?

I searched a lot but did not find anything specific about it.

My problem is how to bind these values into a model?

Thank you for your help

**Comment by joshhanley** [comment]
@asadghazanfary I've not used selectize.js, but I'd recommend making use of Alpine to help with the binding between Livewire and any javascript libraries.

Have a look around the discussions here for other examples, like select2, for how to do it.

Hope this helps!
